src/ray/common/memory_monitor.cc

```diff
@@ -258,24 +137,28 @@ int64_t MemoryMonitor::GetCGroupV2MemoryUsedBytes(const char *stat_path,
   std::string line;
 
   int64_t inactive_file_bytes = kNull;
+  int64_t active_file_bytes = kNull;
   while (std::getline(memstat_ifs, line)) {
     std::istringstream iss(line);
     iss >> title >> value;
-    if (title == kCgroupsV2MemoryStatInactiveKey) {
+    if (title == inactive_file_key) {
       inactive_file_bytes = value;
-      break;
+    } else if (title == active_file_key) {
+      active_file_bytes = value;
     }
   }
 
   int64_t current_usage_bytes = kNull;
   memusage_ifs >> current_usage_bytes;
-  if (current_usage_bytes == kNull || inactive_file_bytes == kNull) {
+  if (current_usage_bytes == kNull || inactive_file_bytes == kNull ||
+      active_file_bytes == kNull) {
     RAY_LOG_EVERY_MS(WARNING, kLogIntervalMs)
-        << "Failed to parse cgroup v2 memory usage. memory.current "
-        << current_usage_bytes << " inactive " << inactive_file_bytes;
+        << "Failed to parse cgroup memory usage. memory usage " << current_usage_bytes
+        << " inactive file " << inactive_file_bytes << " active file "
+        << active_file_bytes;
     return kNull;
   }
-  return current_usage_bytes - inactive_file_bytes;
+  return current_usage_bytes - inactive_file_bytes - active_file_bytes;
```

Comment:
Let's comment why we need to substract `active_file_bytes`:

we can put the link as a good reference: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/resource_management_guide/sec-memory

and comment that 

`total_active_file + total_inactive_file == cache - shmem`

Reply:
and comment that cgroup v2 doesn't have `cache` and `shmem` fields, so we use `total_active_file, total_inactive_file` in formula to unify fomula across v1 / v2